go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/apple/foundationdb/bindings/go v0.0.0-20190411004307-cd5c9d91fad2
 	github.com/armon/go-metrics v0.4.1
 	github.com/armon/go-radix v1.0.0
-	github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef
+	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
```

Comment:
I'm suspicious about there being so many go.mod changes, and it's not clear that they're necessary.  Could you try reverting the go.mod changes, then `go mod tidy`, and seeing what that leaves us with?